### PR TITLE
Returned Keccak hashing and burn BGL changes

### PR DESCRIPTION
**Pull request by van-orton**
### Description
This PR adds some BGL-specific changes that were overwriten by code sync with BTC.
After these changes BGL Node is syncing and validating current BGL blockchain successfully.
@janus please take a look and merge this PR (it's towards your branch btcsyncApril-_2021)

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -181,34 +169,38 @@ class CTestNetParams : public CChainParams {
...
         // Deployment of Taproot (BIPs 340-342)
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1619222400; // April 24th, 2021
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1628640000; // August 11th, 2021
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].min_activation_height = 0; // No activation delay
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1199145601; // January 1, 2008
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1230767999; // December 31, 2008
+
+        // Deployment of Taproot (BIPs 340-342)
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
```
Comment (janus) [inline src/chainparams.cpp:200]: Looks like you repeated `consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT]

Comment (van-orton) [reply src/chainparams.cpp]: Indeed, fixed that, thanks for notifying :+1:

**Comment by van-orton** [comment]
There was also a problem with wallet RPC commands, because older interface wallet.cpp was used from /src/interfaces, that was refactored in BTC so those files were deleted and Makefile changed accordingly, now wallet commands work fine

**Comment by janus** [comment]
> There was also a problem with wallet RPC commands, because older interface wallet.cpp was used from /src/interfaces, that was refactored in BTC so those files were deleted and Makefile changed accordingly, now wallet commands work fine

Awesome.  I almost deleted them, but I was not sure what they were for.

**Pull request merged**